package.json

```diff
@@ -30,7 +30,8 @@
     "lint": "tslint -p . 1>&2",
     "start": "npm run build && run-p \"start:frontend\" \"electron:debug\"",
     "start:frontend": "cross-env USE_FAST_SASS=true BROWSER=none react-scripts start",
-    "build:pack": "electron-builder --dir"
+    "build:pack": "electron-builder --dir",

```

Comment:
'pack' merely creates the runnable directory (so you can test the packaged output), and 'dist' additionally creates the final packaged files (e.g. ZIP). It can be faster turnaround to omit the final packaging if you're diagnosing a problem. In our case, the final package is a ZIP so it's not much overhead; I can remove 'pack' if you'd like. I normally include both because the [electron builder quick start ](https://www.electron.build/#quick-setup-guide)does it.

Reply:
Sounds good, if that's how they have it in the guide then it works with me.